src/events/adapters/importers/fit/importer.fit.mapper.ts

```diff
@@ -106,7 +106,7 @@ export const FITSampleMapper: { dataType: string, getSampleValue(sample: any): n
   {
     dataType: DataPower.type,
     getSampleValue: (sample: any) => {
-      return sample.power || sample.Power;
+      return isNumber(sample.power) ? sample.power : sample.Power;
```

Comment:
@thomaschampagne here was the issue. I think the last sample was 0 and typically JS returned false

Reply:
Nice found! Perfect :)